Source/DafnyCore/Backends/GoLang/GoBackend.cs

```diff
@@ -89,6 +108,14 @@ public class GoBackend : ExecutableBackend {
       }
     }
 
+    // Dafny used to compile to the old Go package system, whereas Go has moved on to a module
+    // system. Although compiler has moved to new system, it still doesn't generate the go.mod file which
+    // is required by go run. Keeping this until we decide if generating go.mod file is the right call.
+    if (GoModuleMode) {
+      outputWriter.WriteLine("go build/run skipped in Go Module Mode");
```

Comment:
This feels like it should be `errorReporter.Info(...)` instead.

Reply:
Changed to use `Reporter.Info`